src/main/kotlin/com/hyperdevs/poeditor/gradle/tasks/ImportPoEditorStringsTask.kt

```diff
@@ -55,6 +56,7 @@ abstract class ImportPoEditorStringsTask
             defaultResPath = extension.defaultResPath.get()
             tags = extension.tags.get()
             languageOverridePathMap = extension.languageValuesOverridePathMap.get()
+            minimumTranslationPercentage = extension.minimumTranslationPercentage.orNull
```

Comment:
If you apply the suggestion that I mentioned in `PoEditorPlugin`, you can just do
```suggestion
            minimumTranslationPercentage = extension.minimumTranslationPercentage.get()
```